source/install/desktop-app-install.rst

```diff
@@ -202,6 +200,16 @@ The following additional documentation resources are also available for the Matt
 Troubleshooting your Desktop App installation
 ----------------------------------------------
 
+Where is configuation stored locally?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The location of the Mattermost desktop app configuration file depends on the platform where you're running Mattermost (and, in the case of macOS, how you've chosen to install the app):
+
+- Windows: ``Users\<username>\AppData\Roaming\Mattermost``
+- macOS standalone download: ``/Users/<username>/Library/Application Support/Mattermost``
```

Comment:
I would go for "macOS installer" (which by definition covers both download and package type, which in this case is DMG).